spec/namespaces/observability.yaml

```diff
@@ -0,0 +1,224 @@
+openapi: 3.1.0
+info:
+  title: OpenSearch Observability Object API
+  description: API for retrieving and managing Observability Objects.
+  version: 1.0.0
+paths:
+  /_plugins/_observability/_local/stats:
+    get:
+      operationId: observability.get_localstats.0
+      x-operation-group: observability.get_localstats
+      x-version-added: '1.1'
```

Comment:
please recheck `x-version-added`. i think it can fix your ci issue

Reply:
@Tokesh committed changes , you can check